### PR TITLE
feat: add verbosity flag to CLI

### LINKED ISSUE #213: Suggestion: Add option to hide skipped files in console output
When running via `'**/*.md' --markdown` and `--skip 'node_modules/'`, there are a LOT of skipped files in the console output. I don't care about any of them and they make it hard to see what good/bad links were checked. The `--silent` option hides good links which I don't want, either - it's nice to be able to confirm which links were checked. I would prefer not to see hundreds/thousands of skipped links - just the links that are being checked and their results. Thank you!

### PR DESCRIPTION
**Pull request by JustinBeckwith**
Fixes #213.  This adds a `--verbosity` flag, which defaults to `WARNING`.  Skipped links now are hidden by default, unless verbosity is set to `INFO` or `DEBUG`.

### CODE REVIEW

**Comment by case** [comment]
🎉🎉🎉

File: src/cli.ts [hunk 1]
```diff
@@ -128,55 +128,78 @@ async function main() {
     serverRoot: flags.serverRoot,
   };
   if (flags.skip) {
-    if (typeof flags.skip === 'string') {
-      opts.linksToSkip = flags.skip.split(' ').filter(x => !!x);
-    } else if (Array.isArray(flags.skip)) {
-      opts.linksToSkip = flags.skip;
-    }
```
Comment (emmenko) [inline src/cli.ts:135]: FYI: removing the support for `skip` to be a list is a breaking change (when using the `linkinator.config.json` config file).

Was this intentional?

**Review by sa-orlov — COMMENTED** [review]
Ik

**Review by sa-orlov — COMMENTED** [review]
Ik

**Review by sa-orlov — COMMENTED** [review]
Ik

File: src/cli.ts (hunk 1 above)
Comment (JustinBeckwith) [reply src/cli.ts:135]: looks like a bug :)

**Review by sa-orlov — COMMENTED** [review]
Ik

**Pull request merged**